tests/lib/state.sh

```diff
@@ -80,6 +88,13 @@ restore_snapd_state() {
 
         tar -C/ -xf "$SNAPD_STATE_FILE"
     fi
+
+    # Start all the units which have to be active
+    for unit in $(cat "$SNAPD_ACTIVE_UNITS"); do
+        if ! systemctl is-active "$unit"; then
+            systemctl start "$unit"
```

Comment:
This fails in the tests when starting `snapd.seeded.service` because both snapd.service and snapd.socket are stopped at the time. Maybe start `snapd.socket` and `snapd.service` first and then the rest?